snykTask/src/task-args.ts

```diff
@@ -72,6 +76,32 @@ class TaskArgs {
       return '';
     }
   }
+
+  public validate() {
+    if (this.failOnThreshold){
+      if (isNotValidThreshold(this.failOnThreshold!)) {
```

Comment:
Non-null assertion is unnecessary here and should generally be avoided, relying on if-statements instead.